src/Modules/Manage/Dnn.PersonaBar.Users/Components/UsersController.cs

```diff
@@ -534,6 +541,15 @@ private static bool IsAdmin(UserInfo user, PortalSettings portalSettings)
                     includeSuperUsers);
         }
 
+        private PortalSettings GetPortalSettings(int portalId)
+        {
+            var portalSettings = new PortalSettings(portalId);
+            var portalAliases = PortalAliasController.Instance.GetPortalAliasesByPortalId(portalId);
+            portalSettings.PrimaryAlias = portalAliases.FirstOrDefault(a => a.IsPrimary);
+            portalSettings.PortalAlias = PortalAliasController.Instance.GetPortalAlias(portalSettings.DefaultPortalAlias);
+            return portalSettings;
+        }
```

Comment:
Looks like code which we could have in other PB commands. If so, then maybe it should be extracted into a common library?

I'd go for an extension method

```c#
public static PortalSettings GetPortalSettings(the IPortalAliasController controller, int portalId)
{
}
```